### PR TITLE
[DOC] Enhanced RDoc for Net:HTTP

### PR DESCRIPTION
**Pull request by BurdetteLamar**
Treats #request_get.

### CODE REVIEW

File: lib/net/http.rb [hunk 1]
```diff
@@ -1090,7 +1090,7 @@ def write_timeout=(sec)
     end
 
     # Returns the continue timeout value.
-    # See Net::HTTP.continue_timeout=.
+    # See Net::HTTP.continue_timeout=.
```
Comment (peterzhu2118) [inline lib/net/http.rb:1093]: Please remove the trailing space.

```suggestion
    # See Net::HTTP.continue_timeout=.
```

File: lib/net/http.rb [hunk 2]
```diff
@@ -1847,30 +1847,28 @@ def trace(path, initheader = nil)
...
+    #   http = Net::HTTP.new(hostname)
+    #   http.request_get('/todos') # => #<Net::HTTPOK 200 OK readbody=true>
     #
-    #     response = http.request_get('/index.html')
-    #     # The entity body is already read in this case.
-    #     p response['content-type']
-    #     puts response.body
+    # With a block given, calls the block with the response object
+    # (with the body _not_ already read),
+    # and returns the response object (_with_ the body already read):
```
Comment (peterzhu2118) [inline lib/net/http.rb:1863]: I don't think talking about whether the body has been read is important. It's an internal state that isn't publicly exposed (except in the output string of `Net::HTTPResponse#inspect`). So I'm not sure what the value of talking about it is.

File: lib/net/http.rb (hunk 1 above)
Comment (BurdetteLamar) [reply lib/net/http.rb]: Fixed.

File: lib/net/http.rb (hunk 2 above)
Comment (BurdetteLamar) [reply lib/net/http.rb]: Removed.

**Review by peterzhu2118 — APPROVED** [review]

**Pull request merged**